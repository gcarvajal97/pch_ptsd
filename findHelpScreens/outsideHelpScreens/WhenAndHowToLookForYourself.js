import React, { Component } from 'react';
import { View, StyleSheet, Text, Image} from 'react-native';
import { translate } from '../../components/translateService';
import colors from '../../colors';

export default class WhenAndHowToLookForYourself extends Component {

  render() {
    return (
      <View>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph1')}</Text>
        <Text style={styles.paragraphTitle}>{translate('whenToLookYourself.paragraphTitle1')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph2')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph3')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph4')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph5')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph6')}</Text>
        <Text style={styles.paragraphTitle}>{translate('whenToLookYourself.paragraphTitle1')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph7')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph8')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph9')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLookYourself.paragraph10')}</Text>
    </View>
    );
  }
}

const styles = StyleSheet.create({
  paragraphTitle: {
    fontFamily: "avenir-heavy",
    fontSize: 20,
    alignSelf: 'stretch',
    textAlign: 'left',
    marginTop: 6,
    marginBottom: 3,
    color: colors.accent
  },
  paragraph: {
    textAlign: 'left',
    fontSize: 14,
    marginVertical: 2
  },
  paragraphBold: {
    textAlign: 'left',
    fontSize: 14,
    fontFamily: "avenir-heavy",
    marginVertical: 2,
    marginLeft: 10
  },
  link: {
    fontSize: 14,
    color: colors.accent,
    textDecorationLine: 'underline'
  }
});