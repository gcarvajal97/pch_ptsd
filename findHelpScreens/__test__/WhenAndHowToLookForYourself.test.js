import React from 'react';
import WhenAndHowToLookForYourself from '../outsideHelpScreens/WhenAndHowToLookForYourself';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<WhenAndHowToLookForYourself/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 