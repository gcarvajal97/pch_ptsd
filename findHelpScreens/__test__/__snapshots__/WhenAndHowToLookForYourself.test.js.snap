// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Sometimes parents need extra help too.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Do you have traumatic stress symptoms that:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Get in the way of getting back to usual activities?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Stop you from enjoying things you used to enjoy?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Seem to be getting worse rather than better?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Have lasted more than a month?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    If so, you may want to seek out a professional who can help you.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Do you have traumatic stress symptoms that:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    When looking for a counselor or therapist to help with traumatic stress symptoms, remember to ask questions such as: 
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Do you have experience working with people after trauma?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -What is your approach to working with this type of problem?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -What is your success rate?
  </Text>
</View>
`;
